pynicotine/gtkgui/ui/search.ui

```diff
@@ -178,6 +178,7 @@
                             <property name="visible">True</property>
                             <property name="has-entry">True</property>
                             <property name="hexpand">True</property>
+                            <property name="id-column">0</property>
```

Comment:
Is this change necessary?

Reply:
It is required for `set_active_id()` to highlight current list item.